lib/orderbook/OrderBook.ts

```diff
@@ -238,23 +246,21 @@ class OrderBook extends EventEmitter {
     });
   }
 
-  private updateOrderQuantity = (order: orders.StampedOrder, decreasedQuantity: number): boolean => {
+  private updateOrderQuantity = (order: orders.StampedOrder, quantityToDecrease: number) => {
```

Comment:
renaming is good, but there are many `decreasedQuantity` usages in other places. should rename them all.

Reply:
Will do, thanks.